hs_file_types/models/base.py

```diff
@@ -123,14 +194,34 @@ def get_add_keyvalue_button():
                     self._get_add_key_value_modal_form()
                     self._get_edit_key_value_modal_forms()
                     self._get_delete_key_value_modal_forms()
-                return root_div
-            else:
-                root_div_extra = div(cls="col-sm-12 content-block", id="filetype-extra-metadata")
-                with root_div_extra:
+            return root_div_extra
+        else:
+            root_div_extra = div(cls="row", id="filetype-extra-metadata")
+            with root_div_extra:
+                with div(cls="col-lg-12 content-block"):
                     legend('Extended Metadata')
                     get_add_keyvalue_button()
                     self._get_add_key_value_modal_form()
-                return root_div
+            return root_div_extra
+
+    def get_temporal_coverage_html_form(self):
+        # Note: When using this form layout the context variable 'temp_form' must be
+        # set prior to calling the template.render(context)
+        root_div = div(cls="row", id="temporal-coverage-filetype")
+        with root_div:
+            with div(cls="col-lg-6 col-xs-12"):
+                with form(id="id-coverage_temporal-file-type", action="{{ temp_form.action }}",
+                          method="post", enctype="multipart/form-data"):
+                    div("{% crispy temp_form %}")
+                    with div(cls="row", style="margin-top:10px;"):
+                        with div(cls="col-md-offset-10 col-xs-offset-6 "
+                                     "col-md-2 col-xs-6"):
+                            button("Save changes", type="button",
+                                   cls="btn btn-primary pull-right",
+                                   style="display: none;",
+                                   onclick="metadata_update_ajax_submit("
+                                           "'id-coverage_temporal-file-type'); return false;")
```

Comment:
@pkdash @aphelionz @mauriel133 Just curious whether this ```onclick=...``` above would count for inline event handler which is not permitted by our stricter security policy.